packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -122,6 +124,7 @@ export default function MainFeedLayout({
   });
   const feedVersion = useFeature(feature.feedVersion);
   const searchVersion = useFeature(feature.search);
+  const isV1Search = searchVersion === SearchExperiment.V1;
```

Comment:
Here you also don't have access to the hook?

Reply:
This is the search experiment, no hook for it